packages/teams-js/src/private/otherAppStateChange.ts

```diff
@@ -0,0 +1,124 @@
+import { sendMessageToParent } from '../internal/communication';
+import { registerHandler, removeHandler } from '../internal/handlers';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
+import { isNullOrUndefined } from '../internal/typeCheckUtilities';
+import { ErrorCode } from '../public';
+import { runtime } from '../public/runtime';
+
+/**
+ * @hidden
+ * @internal
+ * @beta
+ * Limited to Microsoft-internal use
+ *
+ * This capability contains the APIs for handling events that happen to other applications on the host
+ * *while* the developer's application is running. For example, if the developer wants to be notified
+ * when another application has been installed.
+ */
+export namespace otherAppStateChange {
+  /**
+   * v2 APIs telemetry file: All of APIs in this capability file should send out API version v2 ONLY
+   */
+  const otherAppStateChangeTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_2;
+
+  /**
+   * @hidden
+   * @beta
+   * @internal
+   * Limited to Microsoft-internal use
+   *
+   * Represent an event that has happened with other number of applications installed on this host.
+   * (e.g. a new app has been installed)
+   */
+  export interface OtherAppStateChangeEvent {
+    /** An array of app ids that this event applies to */
+    appIds: string[];
```

Comment:
>string [](http://example.com/codeflow?start=12&length=6)

Can this be a...

just kidding.

Reply:
they don't have the emoji I am looking for